.github/workflows/provision.yml

```diff
@@ -14,8 +14,13 @@ jobs:
       AzureTenantId: ${{ secrets.AZURETENANTID }}
       AzureSubscriptionId: ${{ secrets.AZURESUBSCRIPTIONID }}
       AzureClientSecret: ${{ secrets.AZURECLIENTSECRET }}
-      DmxCoreApiUrl: ${{ secrets.DMXCOREAPIURL }}
-      DmxCoreApiAccessKey: ${{ secrets.DMXCOREAPIACCESSKEY }}
+      AzureAdInstance: ${{ secrets.AZUREADINSTANCE }}
+      AzureAdDomain: ${{ secrets.AZUREADDOMAIN }}
+      AzureAdCallbackPath: ${{ secrets.AZUREADCALLBACKPATH }}
+      DmxGatekeeperApiUrl: ${{ secrets.DMXGATEKEEPERAPIURL }}
+      DmxGatekeeperApiAccessKey: ${{ secrets.DMXGATEKEEPERAPIACCESSKEY }}
```

Comment:
@hassanhabib - since we're hooking the Portal up to the Gatekeeper API (instead of Core API), can I update the pipeline secrets accordingly?

Reply:
Yes - we should hook up DMX Portal to DMX Gatekeeper instead of DMX Core.
Which means we need to do the following:
1. Create a new DMX Portal App Registration - call it `DMX-PORTAL-APP-DEV`
2. Create DMX Gatekeeper App Registration - call it `DMX-GATEKEEPER-APP-DEV`
3. Configure `DMX-GATEKEEPER-APP-DEV` to talk to `DMX-CORE-APP-DEV`
4. Configure `DMX-PORTAL-APP-DEV` to talk to `DMX-GATEKEEPER-APP-DEV`
5. Submit a ticket for Admin grants
6. Test everything locally end-to-end 
7. Circle back with the team so we can update our Provision & Infra with the new variables.

let me know if you have any questions.